Knock.net/Resources/Objects/ObjectSubscription.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Knock
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Contains information about a Knock Object record.
+    /// </summary>
+    public class ObjectSubscription
+    {
+
+        /// <summary>
+        /// The object that the recipient is subscribed to
+        /// </summary>
+        [JsonProperty("object")]
+        public Object Object { get; set; }
```

Comment:
Is this right as public Object Object?

Reply:
It works! can use KnockObject if we feel it's best, but I wanted to keep parity with the property name!